blockchain/beacon/Cargo.toml

```diff
@@ -25,5 +25,7 @@ surf                        = { version = "2.3", default-features = false, featu
 tokio                       = { workspace = true, features = ["sync"] }
 
 [dev-dependencies]
-async-std            = { workspace = true, features = ["attributes"] }
+async-std            = { workspace = true, features = ["attributes"], version = "1" }
+quickcheck           = "1"
+quickcheck_macros    = "1"
```

Comment:
```suggestion
async-std            = { workspace = true, features = ["attributes"] }
quickcheck.workspace = true
quickcheck_macros.workspace = true
```

Reply:
This does not seem to be resolved. Did you forget to push a commit?

Reply:
Looks like I did, fixed